website/static/js/osfHelpers.js

```diff
@@ -569,6 +569,48 @@ function humanFileSize(bytes, si) {
     return bytes.toFixed(1) + ' ' + units[u];
 }
 
+function getNodesOriginal(nodeTree, nodesOriginal) {
```

Comment:
This function is the same as https://github.com/CenterForOpenScience/osf.io/pull/4958/files#diff-0edcea532119f03b4ad90b86dc5ba654R618. Why added it again?